src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberMethodSymbol.cs

```diff
@@ -167,6 +167,7 @@ private void MethodChecks(MethodDeclarationSyntax syntax, Binder withTypeParamsB
                 signatureBinder, this, syntax.ParameterList, out arglistToken,
                 allowRefOrOut: true,
                 allowThis: true,
+                addIsConstModifier: IsVirtual || IsAbstract,
```

Comment:
Filed #20053 for that.

---
In reply to: [119902520](https://github.com/dotnet/roslyn/pull/19658#discussion_r119902520) [](ancestors = 119902520)